.ruff.toml

```diff
@@ -0,0 +1,32 @@
+select = [
+    "E",    # pycodestyle
+    "W",    # pycodestyle
+    "F",    # pyflake
+    "I",    # isort
+    "B",    # flake8-bugbear
+    "UP",   # pyupgrade
+]
+
+ignore = [
+    "E501",  # line-too-long
+    "B017",  # pytest.raises(Exception) should be considered evil
+    "B028",  # warnings.warn called without an explicit stacklevel keyword argument
+    "B904",  # check for raise statements in exception handlers that lack a from clause
+]
+
+exclude = [
+    "**/docs",
+]
+
+target-version = "py38"
+
+[per-file-ignores]
+# Ignore unused imports (F401) in these files
+"__init__.py" = ["F401"]
+"graphene_django/compat.py" = ["F401"]
+
+[isort]
```

Comment:
It looks like this should be `[tool.ruff.isort]` since we're using the isort rules via ruff now (https://beta.ruff.rs/docs/settings/#isort), right? It seems ruff has support for each of these options.

Reply:
Ah it's only `[tool.ruff.isort]` if we use `pyproject.toml`. if `ruff.yaml` or `.ruff.yaml` we can omit the `tool.ruff.`. I have checked this. It's working.

Reply:
Ahh gotcha, I hadn't even noticed this was `.ruff.toml` rathr than `pyproject.toml` 👍